osquery/events/events_tests.cpp

```diff
@@ -1,381 +0,0 @@
-/*
```

Comment:
This file was duplicated, we're not removing tests. 
Existing set of tests: https://github.com/facebook/osquery/blob/master/osquery/events/tests/events_tests.cpp